docs/nuget-org/nuget-org-faq.yml

```diff
@@ -60,33 +60,55 @@ sections:
           It is possible to share dependencies between multiple packages, making the total download size for consumers of your NuGet packages smaller.
           
           Dependencies are mostly static and never change. When fixing a bug in code, the dependencies may not need to be updated. If you bundle dependencies, you end up reshipping larger packages every time. By splitting NuGet packages into related dependencies, upgrades are much more fine-grained for consumers of your package.
-          
+
   - name: NuGet.org not accessible
     questions:
-      - question: Why can't I download packages from or upload packages to NuGet.org?
+      - question: What SSL/TLS version and cipher suites does NuGet.org support?
         answer: |
-          First, make sure you're using the latest versions of NuGet. If that version continues to fail, [contact support](https://www.nuget.org/policies/Contact) and provide additional connection troubleshooting information including:
-          
-          - The version of NuGet you're using
+          NuGet.org supports TLS 1.2 and the following cipher suites:
+          - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+          - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+          - TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+          - TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+
+      - question: Why can't I restore packages from NuGet.org?
+        answer: |
+          If you meet transient failures when restoring packages from NuGet.org, we suggest you:
+          - use [V3 API endpoint](https://learn.microsoft.com/nuget/nuget-org/overview-nuget-org#api-endpoint-for-nugetorg) to restore packages from NuGet.org.
+          - use the latest versions of NuGet clients with a better resilience support.
+          - use environment variables of NuGet clients to enhance the retry policy on CI machines.
+
+          | Environment Variable | Description | Remarks |
+          | --- | --- | --- |
+          | NUGET_ENABLE_ENHANCED_HTTP_RETRY | Enables or disables enhanced HTTP retry in NuGet. | Possible values are `true` (default since NuGet CLI 6.2.0) or `false`. |
+          | NUGET_ENHANCED_MAX_NETWORK_TRY_COUNT | Configures the maximum number of times an HTTP connection should be retried when enhanced retries are enabled. | A number representing how many retries to perform, the default value is `6`. |
+          | NUGET_ENHANCED_NETWORK_RETRY_DELAY_MILLISECONDS | Configures the amount of time to wait in milliseconds before retrying an HTTP connection when enhanced retries are enabled. | Number of millseconds to wait, the default value is `1000`. |
+
+          > [!Note]
+          > These environment variables are available since [.NET CLI](https://learn.microsoft.com/nuget/reference/dotnet-commands) (.NET SDK) 6.0.100, [NuGet CLI](https://learn.microsoft.com/nuget/reference/nuget-exe-cli-reference) 6.0.0, Visual Studio 2022 version 17.0 and corresponding MSBuild version.
```

Comment:
@zivkan Could you help me verify that these version numbers and the description here are accurate?

Reply:
It was added in NuGet 6.0: https://github.com/NuGet/NuGet.Client/commit/0a66a9486c69f95ddd5ea57158c86dfb2c9e877f

According to the [NuGet 6.0 release notes](https://learn.microsoft.com/en-us/nuget/release-notes/nuget-6.0), it was available in .NET SDK 6.0 and Visual Studio (msbuild) 17.0.

Reply:
Thanks! @zivkan Could I get an approval from you?

I have checked and followed the release notes. The description has been updated.